src/pretix/plugins/paypal2/static/pretixplugins/paypal2/pretix-paypal.js

```diff
@@ -92,8 +92,8 @@ var pretixpaypal = {
         }
 
         // We are setting the cogwheel already here, as the renderAPM() method might take some time to get loaded.
-        let apmtextselector = $("label[for=input_payment_paypal_apm]");
-        apmtextselector.prepend('<span class="fa fa-cog fa-spin"></span> ');
+        const apmtextselector = $("input[name=payment][value=paypal_apm]").closest("fieldset").find(".provider-name");
```

Comment:
If we at some point nest fieldsets, this selector might be to broad. I’d go to the closest `label`, knowing that the input is inside the label. See comment below regarding `.accordion-panel-label`.

```suggestion
        const apmtextselector = $("input[name=payment][value=paypal_apm]").closest("label").find(".accordion-panel-label");
```

Reply:
```suggestion
        const apmtextselector = $("input[name=payment][value=paypal_apm]").closest("label").find(".accordion-label-text");
```